lib/build.gradle.kts

```diff
@@ -94,33 +92,31 @@ configurations.forEach {
 }
 
 dependencies {
-    implementation("org.jetbrains.kotlin:kotlin-stdlib")
-    coreLibraryDesugaring("com.android.tools:desugar_jdk_libs:2.0.4")
+    implementation(libs.kotlin.stdlib)
+    coreLibraryDesugaring(libs.android.desugaring)
 
-    implementation("androidx.core:core-ktx:1.12.0")
-    api("org.mnode.ical4j:ical4j:${version_ical4j}")
-    implementation("org.slf4j:slf4j-jdk14:2.0.7")       // ical4j logging over java.util.Logger
+    implementation(libs.androidx.core)
+    api(libs.ical4j)
+    implementation(libs.slf4j)       // ical4j logging over java.util.Logger
 
     // ical4j requires newer Apache Commons libraries, which require Java8. Force latest Java7 versions.
-    // noinspection GradleDependency
```

Comment:
Do we need these suppressions? What do they mean? Ignoring vulnerabilities sounds quite hard.

The intention of the original `noinspection GradleDependency` was that Android Studio doesn't suggest updating the libs (which can break the project because newer versions require a Java version that is not available).